CONTRIBUTING.md

```diff
@@ -37,7 +37,7 @@ PR title.
 
 For PRs that you consider ready for review, verify the following locally before you submit it:
 
-* you have a coherent set of logical commits, with messages conforming to the [Conventional Commits](https://github.com/apache/incubator-horaedb/docs/conventional-commit.md) specification;
+* you have a coherent set of logical commits, with messages conforming to the [Conventional Commits](https://github.com/apache/incubator-horaedb-docs/blob/029e993f359907b7502b969a1cffeb8e43cb17fb/docs/src/en/dev/conventional_commit.md) specification;
```

Comment:
I guess this is the link we want: `https://github.com/apache/incubator-horaedb-docs/blob/main/docs/src/en/dev/conventional_commit.md`.